lib/css/modules/_styleTweaks.scss

```diff
@@ -116,3 +116,14 @@ blockquote.twitter-tweet {
 .res-styleTweaks-postTitleCapitalization-lowercase {
 	.entry a.title { text-transform: lowercase; }
 }
+
+.res-styleTweaks-floatingSideBar {
+	body.with-listing-chooser .listing-chooser {
+		position: fixed;
+		top: 0;
+
+		.contents {
+			margin-top: 65px;
```

Comment:
Note: reddit hardcodes `65px` in [their own CSS](https://github.com/reddit/reddit/blob/3ea40a09cc38f3d62fc008cf2ba3f858f638b755/r2/r2/public/static/css/reddit.less#L9810) (and I don't think they'll change the header height out-of-the-blue), so I'm not concerned about hardcoding this.